datachannel_js.go

```diff
@@ -20,9 +20,10 @@ type DataChannel struct {
 
 	// Keep track of handlers/callbacks so we can call Release as required by the
 	// syscall/js API. Initially nil.
-	onOpenHandler    *js.Func
-	onCloseHandler   *js.Func
-	onMessageHandler *js.Func
+	onOpenHandler       *js.Func
+	onCloseHandler      *js.Func
+	onMessageHandler    *js.Func
+	onBufferedAmountLow *js.Func
```

Comment:
Optionally, we could call this `onBufferedAmountLowHandler` to more closely match the other functions we are tracking.